src/main/java/competition/subsystems/lights/LightsCommunicationSubsystem.java

```diff
@@ -132,7 +132,7 @@ public void periodic() {
             }
         } else if (collector.getGamePieceCollected()) {
             currentState = LightsStateMessage.GamePieceCollected;
-        } else if(arm.isMaintainerAtGoal() && arm.upperArm.getArmPositionInDegrees() > 20){
+        } else if(arm.isMaintainerAtGoal() && arm.getCurrentXZCoordinates().x > 20){
```

Comment:
is 20 still the right number in x co-ordinates? what were the x co-ordinates of the high and mid points we were setting up here? (the numbers in the code were the angles). I just want to double check that this won't be true unless we're really out near a scoring point

Reply:
I'm not sure, I only placed 20 in there as a place holder. Perhaps we can test this in the pit tomorrow?